src/serlio/modifiers/PRTModifierAction.h

```diff
@@ -58,6 +58,7 @@ class PRTModifierEnum {
 	MDoubleArray mFVals;
 	MIntArray mBVals;
 	bool mRestricted = true;
+	MString mValuesAttr = "";

```

Comment:
It is not necessary to initialize by assignment to empty string. This is already done by the default constructor.

Reply:
removed initialization